repology/parsers/parsers/debian.py

```diff
@@ -131,6 +131,10 @@ def _extra_handling(self, pkg: PackageMaker, pkgdata: dict[str, str]) -> None:
             pkg.add_name(pkgdata['SourceName'], NameType.OPENWRT_SOURCENAME)
         pkg.set_arch(pkgdata['Architecture'])
         pkg.set_extra_field('path', '/'.join(pkgpath[2:]))
+        pkg.add_homepages(pkgdata.get('URL'))
+        if cpeid := pkgdata.get('CPE-ID'):
+            cpe_components = cpeid.split(':')
+            pkg.add_cpe(cpe_components[3], cpe_components[4])
```

Comment:
Shouldn't that be 2 and 3, instead of 3 and 4?

Reply:
So the line is `CPE-ID: cpe:/a:netfilter:iptables`. If the parsing works right the content of `CPE-ID` should be `cpe:/a:netfilter:iptables`, which actually changes the right values to be 1 and 2, right?

Reply:
No, why? [0] is `cpe`, [1] is `/a`, [2] is `netfilter` and [3] is `iptables`. We need 2 and 3.